e2e/_suites/kubernetes-autodiscover/features/beats.feature

```diff
@@ -0,0 +1,52 @@
+@beats
+Feature: Beats
+  Use Kubernetes autodiscover features in Beats to monitor pods
+
+Scenario: Pod is started
+  Given a cluster is available
```

Comment:
We could remove the `Given` clause and set it as part of the setUp, performed in the beforeScenario test hook. If it does not matter to create/destroy the cluster among scenarios, we could use the beforeSuite hook

Reply:
Mmm I see, this is only a check that the cluster is up, right? Then I'm OK with this one

Reply:
I think we could simply remove the `Given` clause and assert it in the `beforeSuite` or `beforeScenario` hook. From a high level perspective, I would expect the cluster to be available as a strong requirement to run the tests. We usually create and start runtime (test-time) dependencies in the proper `beforeSuite` hook, exiting if they cannot be satisfied.

Reply:
This is a good point, I added it at the beginning to setup the cluster but it actually doesn't contribute anything to the scenario, at least by now, scenario is setup in the `BeforeScenario` hook as you propose. I will remove this step.